Packs/SpaceX/Integrations/SpaceX/SpaceX.yml

```diff
@@ -0,0 +1,2912 @@
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.norad_id
+      description: Norad ID
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit
+      description: Orbit
+      type: string
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.payload_type
+      description: Payload type
+      type: string
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.payload_id
+      description: Payload ID
+      type: string
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.reused
+      description: Reused
+      type: boolean
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.manufacturer
+      description: Manufacturer
+      type: string
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.payload_mass_kg
+      description: Payload mass (kg)
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.customers
+      description: Customers
+      type: unknown
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.arg_of_pericenter
+      description: Arg of pericenter
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.reference_system
+      description: Reference system
+      type: string
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.eccentricity
+      description: Eccentricity
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.epoch
+      description: Epoch
+      type: date
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.mean_anomaly
+      description: Mean anomaly
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.mean_anomaly
+      description: Inclination degree
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.semi_major_axis_km
+      description: Semi major axis (km)
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.regime
+      description: Regime
+      type: string
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.longitude
+      description: Longitude
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.mean_motion
+      description: Mean motion
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.apoapsis_km
+      description: Apoapsis (km)
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.period_min
+      description: Period (mins)
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.raan
+      description: Raan
+      type: number
+    - contextPath: SpaceX.Launches.rocket.second_stage.payloads.orbit_params.lifespan_years
```

Comment:
We advise to use context outputs path of length <= 5. If possible let's make it of length 5. This comment is for all outputs in the yaml.